src/storage/v2/indices/text_index.cpp

```diff
@@ -78,6 +79,34 @@ nlohmann::json TextIndex::SerializeProperties(const std::map<PropertyId, Propert
   return serialized_properties;
 }
 
+std::string TextIndex::CopyPropertyValuesToString(const std::map<PropertyId, PropertyValue> &properties) {
+  std::vector<std::string> indexable_properties_as_string;
```

Comment:
This is a good idea, but not 100% suitable here because certain types (e.g. temporal) that can be meaningfully converted to strings aren’t added to text indices, and so they shoudn’t appear here either (see [text_index.cpp#L98](https://github.com/memgraph/memgraph/blob/7b320c15d0c0d6289e6a4fd5a6874722278aeaa6/src/storage/v2/indices/text_index.cpp#L98).